### PR TITLE
Connected the account name change and the password change to the backend

### LINKED ISSUE #169: Connecting the profile page to the backend
**Is your feature request related to a problem? Please describe.**
Currrently the profile page is not connected to the backend

**Describe the solution you'd like**
Do the same

**Additional context**
Are you working on this?
Yes

### PR DESCRIPTION
**Pull request by ritwizsinha**
Fixes #169 
Changes: Connected the change of username and password in the profile page to the backend

Surge Deployment Link: https://pr-170-sahaayak.surge.sh

### CODE REVIEW

File: src/components/Settings/Password.js [hunk 1]
```diff
@@ -1,23 +1,51 @@
...
+  }
+  onChange = e=>{
+    e.preventDefault();
+    this.setState({
+      [e.target.name]: e.target.value
+    })
+  }
+  onClick = ()=>{
+    if(this.state.newPassword !== this.state.confNewPassword){
+      message.error("New password doesn't match");
```
Comment (harsh-jindal) [inline src/components/Settings/Password.js]: Change this to `Password does not match`

Comment (harsh-jindal) [reply src/components/Settings/Password.js]: Reset password is not working. Also no need to have an ongoing redux actions, set a loading state on the button as we have in login. Also use .then() to set the message to display, have a separate state for that.

Comment (ritwizsinha) [reply src/components/Settings/Password.js]: It is working fine for me , made the other change

File: src/components/Settings/Password.js [hunk 2]
```diff
@@ -1,12 +1,45 @@
...
+    })
+  }
+  openNotificationWithIcon = (type,message) => {
+    notification[type]({
+      message: message,
+    });
+  };
+  onClick = ()=>{
+    if(this.state.newPassword !== this.state.confNewPassword){
+      return this.openNotificationWithIcon('error','Password doesn\'t match');
```
Comment (harsh-jindal) [inline src/components/Settings/Password.js:26]: `doesn\'t` Fix this in your next PR

**Review by harsh-jindal — APPROVED** [review]
@ritwizsinha

**Pull request merged**